Paper.tex

```diff
@@ -1891,7 +1898,7 @@ \section{Signing Transactions}\label{app:signing}
 \begin{eqnarray}
 \mathtt{ECDSAPUBKEY}(p_{\mathrm{r}} \in \mathbb{B}_{32}) & \equiv & p_{\mathrm{u}} \in \mathbb{B}_{64} \\
 \linkdest{ECDSASIGN}\mathtt{ECDSASIGN}(e \in \mathbb{B}_{32}, p_{\mathrm{r}} \in \mathbb{B}_{32}) & \equiv & (v \in \mathbb{B}_{1}, r \in \mathbb{B}_{32}, s \in \mathbb{B}_{32}) \\
-\mathtt{ECDSARECOVER}(e \in \mathbb{B}_{32}, v \in \mathbb{B}_{1}, r \in \mathbb{B}_{32}, s \in \mathbb{B}_{32}) & \equiv & p_{\mathrm{u}} \in \mathbb{B}_{64}
+\mathtt{ECDSARECOVER}(e \in \mathbb{B}_{32}, v \in \mathbb{B}_{2}, r \in \mathbb{B}_{32}, s \in \mathbb{B}_{32}) & \equiv & p_{\mathrm{u}} \in \mathbb{B}_{64}
```

Comment:
This is more accurate because the domain of the recovery identifier is {0, 1, 2, 3} (i.e. B_2), but we explicitly prohibit values 2 and 3 that represent infinite values.

Reply:
But `B` is in bytes, so 1 byte would fit [0,3]?

Reply:
Ah, yes, you're right, it's in bytes. My bad!

Reply:
Could use `N_2` if you want to complicate it? But then conversion needs to be defined somewhere I guess 😓